commands/webhook.go

```diff
@@ -0,0 +1,472 @@
+package commands
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/mattermost/mattermost-server/model"
+	"github.com/mattermost/mmctl/client"
+	"github.com/mattermost/mmctl/printer"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+type StoreResult struct {
+	Data interface{}
+	Err  *model.AppError
+}
+
+var WebhookCmd = &cobra.Command{
+	Use:   "webhook",
+	Short: "Management of webhooks",
+}
+
+var WebhookListCmd = &cobra.Command{
```

Comment:
For consistency with the rest of the naming, we should use `ListWebhookCmd` (the function name is correct tho)